test/MathBuiltins/log/float2_log1p.cl

```diff
@@ -0,0 +1,19 @@
+// RUN: clspv %s -o %t.spv
```

Comment:
Sorry for the hassle, but we're trying to move to targeted tests using `clspv-opt` where possible. Could you please convert these to be based on LLVM IR and just running ReplaceOpenCLBuiltins? The tests look correct.

Reply:
Sure. I'm on vacation this week but I'll update the tests and get the  CLA sorted with our POC at the start of next week.

Could you point me to an example of a test that's in the correct new style?

Reply:
Sure, take a look at the tests in `test/MathBuiltins/isfinite` (e.g. [isfinite_half2.ll](https://github.com/google/clspv/blob/master/test/MathBuiltins/isfinite/isfinite_half2.ll)).

Reply:
I've changed the tests to LLVM IR.